Remotion/ObjectBinding/Web/UI/Controls/BocEnumValueImplementation/Rendering/BocEnumValueRenderer.cs

```diff
@@ -167,10 +167,13 @@ private ListControl GetListControl (BocEnumValueRenderingContext renderingContex
 
       var isRadioButtonList = renderingContext.Control.ListControlStyle.ControlType == ListControlType.RadioButtonList;
       var isDropDownList = renderingContext.Control.ListControlStyle.ControlType == ListControlType.DropDownList;
+      var isListBox = renderingContext.Control.ListControlStyle.ControlType == ListControlType.ListBox;
```

Comment:
two things;
this change gets to be be moved into a stand alone bugfix in https://re-motion.atlassian.net/browse/RM-7332 on develop. Here in the branch, we accept that the disabled behavior doesn't work until we have rebased.

And I looked again into the asp.net source code and found a cleaner solution: in `ListControlStyle.Create()`, we create a derived version of ListBox (name: `CompatibleListBox`, nested type in Style where the `SupportsDisabledAttribute` is overridden to return `true`.) this way, things are fixed closer at the root-cause instead of worked-around.